modules/assisted-installer-how-to-use-the-assisted-installer.adoc

```diff
@@ -0,0 +1,44 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id="how-to-use-the-assisted-installer_{context}"]
+= How to use the Assisted Installer
+
+The {product-title} link:https://console.redhat.com/openshift/assisted-installer/clusters/~new[{ai-full}] is a user-friendly installation solution offered on the link:http://console.redhat.com[Red Hat Hybrid Cloud Console]. The {ai-full} supports the various deployment platforms with a focus on bare metal.
+
+The {ai-full} provides installation functionality as a service. This software-as-a-service (SaaS)  approach has the following advantages:
+
+* *Web user interface:* The web user interface performs cluster installation without the user having to create the installation configuration files manually.
+* *No bootstrap node:* A bootstrap node is not required when installing with the {ai-full}. The bootstrapping process executes on a node within the cluster.
+* *Hosting:* The {ai-full} hosts:
+  - Ignition files
+  - The installation configuration
+  - A discovery ISO
+  - The installer
+* *Simplified deployment:* Deployment does not require in-depth knowledge of {product-title}. The {ai-full} provides reasonable defaults. Additionally, the {ai-full} performs rich pre-flight validation before installation to catch configuration-related installation errors before they occur.
+* *Advanced networking:* The {ai-full} supports IPv4/IPv6 dual stack networking, NMState-based static IP addressing, and an HTTP/S proxy.
+* *REST API:* The {ai-full} has a REST API, enabling automation.
```

Comment:
Do you want to add some links?
- This is the swagger that defines the API: https://github.com/openshift/assisted-service/blob/master/swagger.yaml
- Doc on how to use it: https://github.com/openshift/assisted-service/blob/master/docs/cloud.md

Reply:
They ask us not to link to github docs. So we'd probably just bring these downstream at some point.